src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4090,7 +4089,6 @@ void SizeAirLoopBranches(EnergyPlusData &state, int const AirLoopNum, int const
                               format("Primary air system volumetric flow rate = {:.4R} m3/s.", PrimaryAirSystems(AirLoopNum).DesignVolFlowRate));
             ShowContinueError(state, "Check flow rate inputs for components in this air loop and,");
             ShowContinueError(state, "if autosized, check Sizing:Zone and Sizing:System objects and related inputs.");
-            ShowFatalError(state, "Previous condition causes termination.");
```

Comment:
remove the FatalError so that the simulation can continue.  Allowing the simulation to continue with 0 flow for the AirLoop is consistent with the way 0 flow at the PlantLoop is currently handled

Reply:
I agree with this. It's never good to stop the simulation just because loads were 0.

Reply:
I still feel like if we are handling the problem and continuing the simulation, that itself is a warning, not an error.  But I won't hold this up for that.  I agree that continuing the simulation itself is an improvement.